src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Parser/SPDX30Parser.cs

```diff
@@ -157,7 +163,299 @@ public SPDX30Parser(
         return null;
     }
 
-    public Spdx22Metadata GetMetadata()
+    private void ValidateContext(ContextsResult result)
+    {
+        if (result.Contexts == null || !result.Contexts.Any() || result.Contexts.Count() > 1)
+        {
+            throw new ParserException($"The context property is either empty or has more than one string.");
+        }
+    }
+
+    /// <summary>
+    /// Converts JSON objects to SPDX elements.
+    /// </summary>
+    /// <param name="jsonList"></param>
+    /// <returns></returns>
+    public ElementsResult ConvertToElements(List<object>? jsonList, ref ParserStateResult? result, string? requiredComplianceStandard, IReadOnlyCollection<string>? entitiesWithDifferentNTIARequirements)
```

Comment:
This doesn't seem to be called externally--could it be private? I'm also confused why `requiredComplianceStandard` is passed in--isn't is already set in the constructor?

Reply:
I've made them private and removed redundant params.